### PR TITLE
add softplus to jax

### PR DESCRIPTION
**Pull request by PatricYan**
- [ ] #3030 
1. add **softplust** to **jax** 
2. test case

### CODE REVIEW

**Review by ankitdsi2010 — CHANGES_REQUESTED** [review]
fix the error in the [test-frontend-jax test](https://github.com/unifyai/ivy/runs/7835875962?check_suite_focus=true#step:4:131)

**Comment by PatricYan** [comment]
Hi, it is ** test_jax_nn_gelu** error for the backended code updated, I think after this PR, I can make a new PR for this error

**Comment by PatricYan** [comment]
> fix the error in the [test-frontend-jax test](https://github.com/unifyai/ivy/runs/7835875962?check_suite_focus=true#step:4:131)

done, the test case error is fixed

File: ivy_tests/test_ivy/test_frontends/test_jax/test_jax_nn_activations.py [hunk 1]
```diff
@@ -42,3 +42,35 @@ def test_jax_nn_softmax(
...
+        input_dtypes=input_dtype,
+        as_variable_flags=as_variable,
+        with_out=False,
+        num_positional_args=num_positional_args,
+        native_array_flags=native_array,
+        fw=fw,
+        frontend="jax",
+        fn_tree="nn.softplus",
+        x=np.asarray(x, dtype=input_dtype),
+    )
```
Comment (ankitdsi2010) [inline ivy_tests/test_ivy/test_frontends/test_jax/test_jax_nn_activations.py]: Add newline at end of file

**Review by ankitdsi2010 — CHANGES_REQUESTED** [review]

File: ivy_tests/test_ivy/test_frontends/test_jax/test_jax_nn_activations.py (hunk 1 above)
Comment (PatricYan) [reply ivy_tests/test_ivy/test_frontends/test_jax/test_jax_nn_activations.py]: done

**Comment by PatricYan** [comment]
Hi, I have updated the code for the conflict. and in test-fronted-jax it is ** test_jax_nn_softmax** not **test_jax_nn_softplus** error

**Comment by PatricYan** [comment]
Hi @ankitdsi2010, would you like to make a review for it?

**Comment by ankitdsi2010** [comment]
Thanks for your contribution

**Pull request merged**